SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    
+    var newConversationButton: UIButton!
+    var feedbackLabel: UILabel!
+    var searchController: UISearchController! // TODO: For later
+    var feedbackTableView: UITableView!
+        
+    var feedbackData: [Feedback]!
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    var countEditTaps: Int = 0
+    var isFeedback: Bool!
+    let cellHeight: CGFloat = 75
+    let reuseID: String = "FeedbackTableViewCell"
```

Comment:
Also for a constant like `reuseIdentifier` I think we can just either stick this into a private `Constants` enum within this class or put it as a static variable of the cell that it corresponds to. I personally lean towards the latter because if we wanna reuse the cell we don't have to redeclare the `reuseIdentifier` several times. @kevinchan159 Thoughts on this too?

Reply:
i agree